.github/workflows/docs.yaml

```diff
@@ -24,7 +24,7 @@ on:
 jobs:
   build_docs:
     name: Documentation
-    runs-on: 'macos-latest'
+    runs-on: 'macos-14'
```

Comment:
I don't think this gave a significant improvement in speed.

Reply:
This could be because this error:

```
 TypeError: argument of type 'URL' is not iterable
Process Process-567:
Traceback (most recent call last):
  File "/Users/runner/miniconda3/envs/test-environment/lib/python3.11/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/Users/runner/miniconda3/envs/test-environment/lib/python3.11/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/runner/miniconda3/envs/test-environment/lib/python3.11/site-packages/nbsite/pyodide/__init__.py", line 231, in _execution_process
    js, js_exports, js_modules, css, global_exports = extract_extensions(code)
                                                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/miniconda3/envs/test-environment/lib/python3.11/site-packages/nbsite/pyodide/__init__.py", line 132, in extract_extensions
    js += [bundle.cdn_url for bundle in extensions if bundle.cdn_url and
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/miniconda3/envs/test-environment/lib/python3.11/site-packages/nbsite/pyodide/__init__.py", line 133, in <listcomp>
    '@holoviz/panel@' not in bundle.cdn_url]
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: argument of type 'URL' is not iterable
```

Reply:
With the fix in nbsite the docs build finish in just under an hour.

[3502e2c](https://github.com/holoviz/panel/pull/6369/commits/3502e2c7720c9d673eb8e0fa13f7dc05fc1ade0d) - https://github.com/holoviz/panel/actions/runs/7979403809